scripts/rmc.py

```diff
@@ -277,23 +275,28 @@ def get_config(option):
                 process.stdout))
         return process.stdout
 
-    rustflags = rustc_flags(mangler, symbol_table_passes, restrict_vtable) + get_config("--rmc-flags").split()
+    rustflags = rustc_flags(extra_args.mangler, symbol_table_passes,
+                            extra_args.restrict_vtable) + get_config("--rmc-flags").split()
     rustc_path = get_config("--rmc-path").strip()
-    build_cmd = ["cargo", "build", "--target-dir", str(target_dir)]
-    if build_target:
-        build_cmd += ["--target", str(build_target)]
+    cargo_cmd = ["cargo", "test", "--no-run"] if extra_args.tests else ["cargo", "build"]
```

Comment:
What about flipping the commands since `cargo build` is the default?

```suggestion
    cargo_cmd =  ["cargo", "build"] if not extra_args.tests else ["cargo", "test", "--no-run"]
```